src/modules/controllib/uorb/blocks.cpp

```diff
@@ -83,16 +84,16 @@ void BlockWaypointGuidance::update(vehicle_global_position_s &pos,
 BlockUorbEnabledAutopilot::BlockUorbEnabledAutopilot(SuperBlock *parent, const char *name) :
 	SuperBlock(parent, name),
 	// subscriptions
-	_att(ORB_ID(vehicle_attitude), 20, &getSubscriptions()),
-	_attCmd(ORB_ID(vehicle_attitude_setpoint), 20, &getSubscriptions()),
-	_ratesCmd(ORB_ID(vehicle_rates_setpoint), 20, &getSubscriptions()),
-	_pos(ORB_ID(vehicle_global_position), 20, &getSubscriptions()),
-	_missionCmd(ORB_ID(position_setpoint_triplet), 20, &getSubscriptions()),
-	_manual(ORB_ID(manual_control_setpoint), 20, &getSubscriptions()),
-	_status(ORB_ID(vehicle_status), 20, &getSubscriptions()),
-	_param_update(ORB_ID(parameter_update), 1000, &getSubscriptions()), // limit to 1 Hz
+	_att(ORB_ID(vehicle_attitude), 20, 0, &getSubscriptions()),
+	_attCmd(ORB_ID(vehicle_attitude_setpoint), 20, 0, &getSubscriptions()),
+	_ratesCmd(ORB_ID(vehicle_rates_setpoint), 20, 0, &getSubscriptions()),
+	_pos(ORB_ID(vehicle_global_position), 20, 0, &getSubscriptions()),
+	_missionCmd(ORB_ID(position_setpoint_triplet), 20, 0, &getSubscriptions()),
+	_manual(ORB_ID(manual_control_setpoint), 20, 0, &getSubscriptions()),
+	_status(ORB_ID(vehicle_status), 20, 0, &getSubscriptions()),
+	_param_update(ORB_ID(parameter_update), 1000, 0, &getSubscriptions()), // limit to 1 Hz
 	// publications
-	_actuators(ORB_ID(actuator_controls_0), &getPublications())
+	_actuators(ORB_ID(actuator_controls_0), -1, &getPublications())
```

Comment:
Or, if this my ignorance and this is the default priority, can we make this a constexpr in the publication class so its clear what this is trying to do?